src/get-started/setup-an-ide.md

```diff
@@ -107,3 +109,30 @@ The third config is `unit test`, which enables you can run/debug unit tests. You
 ![unit test output](../img/unit-test-output.png)
 
 If you encounter any problems during your journey, do not hesitate to reach out on the [TiDB Internals forum](https://internals.tidb.io/).
+
+## Visual Studio Code
+
+VS Code is a generic IDE that has good extensions for working with Go and TiDB.
+
+![VS Code with TiDE](../img/vscode_tide.png)
+
+### Prerequisites
+
+* `go`: TiDB is a Go project thus its building requires a working `go` environment. See the previous [Install Golang](install-golang.md) section to prepare the environment.
+* TiDB source code: See the previous [Get the code, build and run](build-tidb-from-source.md) section to get the source code.
+
+### Download VS Code
+
+Download GoLand from [here](https://code.visualstudio.com/Download) and install it.
+
+Now install these extensions:
+
+* [Go](https://marketplace.visualstudio.com/items?itemName=golang.Go)
+* [TiDE](https://marketplace.visualstudio.com/items?itemName=dragonly.ticode)
```

Comment:
And I think TiDE is also not mandatory? 

Also, GoLand instructions above mainly focus on simple tasks like unistore and unit tests. But TiDE is targeted at more complex tasks (and doesn't support unistore)?

Reply:
Yes not mandatory, but quite useful. I think users would be smart enough to filter this out if needed and if they don't that's also not really a problem.